dockers/docker-fpm-frr/start.sh

```diff
@@ -34,7 +34,7 @@ rm -f /var/run/rsyslogd.pid
 supervisorctl start rsyslogd
 
 # start eoiu pulling, only if configured so
-if [[ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu') == 'true' ]]; then
+if [[ ! -z $(sonic-cfggen -d -v 'WARM_RESTART') ]] && [[ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu') == 'true' ]]; then
```

Comment:
Would it better to fix sonic-cfggen to fix the issue?
Also in your fix what if WARM_RESTART.bgp wasn't created?

Reply:
Agreed and fixed in sonic-cfggen.